fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceTableGroup.java

```diff
@@ -679,12 +679,12 @@ public void addComposites(List<IDatabaseObject> group, String prefix) {
                 }
 
                 statements.add(new DropForeignKeyConstraint(schemaName, tableName,
-                        "FK_OBSERVATION_COMPOSITES_DATE",
-                        "FK_OBSERVATION_COMPOSITES_LATLNG",
-                        "FK_OBSERVATION_COMPOSITES_NUMBER",
-                        "FK_OBSERVATION_COMPOSITES_QUANTITY",
-                        "FK_OBSERVATION_COMPOSITES_STR",
-                        "FK_OBSERVATION_COMPOSITES_TOKEN"));
+                        FK + tableName + _STR,
+                        FK + tableName + _NUMBER,
+                        FK + tableName + _DATE,
+                        FK + tableName + _TOKEN,
+                        FK + tableName + _QUANTITY,
+                        FK + tableName + _LATLNG));
```

Comment:
great catch! seems we have never noticed this before!!

Reply:
and it could be nicer if we can keep the order for the FKs - still from _D to _T

Reply:
i didn't quite follow the order comment; i reordered these so that the order matches the order in the statements above it (in the same method)...did you want something different?

Reply:
I mean in the original codes, they are alphabet ordered.